back/lib/participation_method/community_monitor_survey.rb

```diff
@@ -62,5 +57,48 @@ def validate_phase
     def supports_event_attendance?
       false
     end
+
+    def supports_custom_field_categories?
+      true
+    end
+
+    private
+
+    def page_field(custom_form, key)
+      CustomField.new(
+        id: SecureRandom.uuid,
+        key: key,
+        resource: custom_form,
+        input_type: 'page',
+        page_layout: 'default'
+      )
```

Comment:
Should we include the Category Label as the Page Title? :thinking:

Reply:
Will include this for now, but let's see how it looks and we can always remove